Plugins/Detray/src/DetrayGeometryConverter.cpp

```diff
@@ -63,7 +62,7 @@ detray::io::surface_payload Acts::DetrayGeometryConverter::convertSurface(
              : (surface.geometryId().sensitive() > 0
                     ? detray::surface_id::e_sensitive
                     : detray::surface_id::e_passive));
-  surfacePayload.mask = convertMask(surface.bounds());
+  surfacePayload.masks = {convertMask(surface.bounds())};
```

Comment:
Do I understand correctly, that the surfaces will remain split and the vector contains always one element?

Reply:
`DetrayGeometryConverter` is the Gen2 conversion code, which I don't think we'll update.
For my (still wip) Gen3 conversion branch, I'm keeping them split for right now until the conversion works correctly, and then I'll update this. 

Does that make sense?